plugin-management-library/pom.xml

```diff
@@ -55,6 +55,13 @@
             <artifactId>snakeyaml</artifactId>
             <version>1.24</version>
         </dependency>
+        <!-- https://mvnrepository.com/artifact/com.github.stefanbirkner/system-rules -->		<dependency>
+		    <groupId>com.github.stefanbirkner</groupId>
+		    <artifactId>system-rules</artifactId>
```

Comment:
Properly better to use `system-lambda`

cc @stefanbirkner

Reply:
You're right. I added system-lambda in #123 to plugin-management-library.